app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreenViewModel.kt

```diff
@@ -90,6 +92,10 @@ class DebugScreenViewModel
         logFileWriter.deleteAllLogFiles()
     }
 
+    fun restartSlowSyncForRecovery() = viewModelScope.launch {
+        restartSlowSyncProcessForRecoveryUseCase()
+    }
```

Comment:
```suggestion
   fun restartSlowSyncForRecovery() {
        viewModelScope.launch { 
            restartSlowSyncProcessForRecovery() 
        }
    }
```

This is just a suggestion, since we (almost never) do make use of the returned `Job` from the launch method on Composables/Screen views, so just Unit should do it :D